ProjectSettings/ProjectSettings.asset

```diff
@@ -580,7 +580,7 @@ PlayerSettings:
   m_RenderingPath: 1
   m_MobileRenderingPath: 1
   metroPackageName: Microsoft.MixedReality.Toolkit
-  metroPackageVersion: 2018.9.0.0
+  metroPackageVersion: 2.0.0.0
```

Comment:
Is there any way to differentiate this as a beta?

Reply:
this is the UWP version number. to the best of my knowledge, it must be all integers